package.json

```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "scripts": {
     "start": "node server.js",
-    "test": "mocha"
+    "test": "mocha --exit"
```

Comment:
Not sure why this is showing up in the diff -- master already has the `--exit` flag.

Reply:
It must be showing a diff of what changed before vs after within my branch, not my branch vs master. Does that make sense? My best guess anyway.

Reply:
I forked master before you added the exit flag and bumped the mocha version, then I made those same changes in my branch.